source/support.html.erb

```diff
@@ -134,7 +138,24 @@ title: Support
 
       <p>
         Outside of working hours, please use the escalation contact
-        details you have been sent.
+        details you have been sent.</p>
+
+        <h2 class="heading-large">Keep up to date</h2>
```

Comment:
Perhaps change to "Keeping up to date" just to make consistent with the previous heading "Contacting us..."